tests/test_physics/test_bargmann/test_bargmann_repr.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+
+from mrmustard import math
+from mrmustard.lab import Attenuator, Coherent, Gaussian, Ggate
+from mrmustard.physics import representations
+from mrmustard.physics.bargmann import contract_two_Abc, reorder_abc, wigner_to_bargmann_rho
+from mrmustard.physics.representations import Bargmann
+from tests.random import random_Ggate, single_mode_unitary_gate, n_mode_mixed_state, Abc_triple
+from hypothesis import given
+
+
+def test_make_cat():
+    cat = Bargmann(*Coherent(1.0).bargmann()) + Bargmann(*Coherent(-1.0).bargmann())
+    assert np.allclose(cat.A[0], cat.A[1])
+    assert np.allclose(cat.b[0], -cat.b[1])
+
+
+def test_muldiv():
```

Comment:
I just don't understand what this test function tests for.

Reply:
oops, I had left the bottom part of the older version of this test, thanks!